src/mame/drivers/hp80.cpp

```diff
@@ -157,12 +160,13 @@ void hp85_state::machine_start()
 	m_video_mem.resize(VIDEO_MEM_SIZE);
 
 	// ROM in bank 0 is always present (it's part of system ROMs)
-	membank("rombank")->configure_entry(0 , m_rom00);
+	m_rombank = membank("rombank");
```

Comment:
If you're doing a tagmap lookup here, errors can't be caught at validation time.